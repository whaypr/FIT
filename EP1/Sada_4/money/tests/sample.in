100.80
500 1x
200 3x
1.00 10x
0.20 2x
-1
500 10x
200 12x
100 8x
0.10 1x
0.20 0x
0.50 100x
20 2x
-1
200
10 19x
-1
200 1x
-1
-1