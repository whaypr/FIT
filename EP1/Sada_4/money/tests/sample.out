7 tenders must be exchanged.
The payment is impossible.
